js/main.js

```diff
@@ -1,90 +1,24 @@
-  },
-  {
-    id: '1d88fefe-edf1-4cda-844a-babbf29bb2b3',
-    name: 'Villa Kunerad',
-    city: 'Vysokie Tatry',
-    country: 'Slowakia',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/villa-kunerad_gdbqgv.jpg',
-  },
-  {
-    id: 'a2bf824d-edd8-41f0-8b70-244334086ab4',
-    name: 'Hostel Friendship',
-    city: 'Berlin',
-    country: 'Germany',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379364/fe/hostel-friendship_aw6tn7.jpg',
-  },
-  {
-    id: '4024535d-a498-4274-b7cb-f01ada962971',
-    name: 'Radisson Blu Hotel',
-    city: 'Kyiv',
-    country: 'Ukraine',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/radisson-blu-hotel_jwtowg.jpg',
-  },
-  {
-    id: 'e51e71f6-6baf-4493-b3ae-25dc27cdc238',
-    name: 'Paradise Hotel',
-    city: 'Guadalupe',
-    country: 'Mexico',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/paradise-hotel_i6whae.jpg',
-  },
-  {
-    id: '87d2b966-2431-43f3-8c0d-2c8723474dfc',
-    name: 'Hotel Grindewald',
-    city: 'Interlaken',
-    country: 'Switzerland',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/hotel-grindewald_zsjsmy.jpg',
-  },
-  {
-    id: '190221c6-b18f-4dba-97de-e35f0e14c023',
-    name: 'The Andaman Resort',
-    city: 'Port Dickson',
-    country: 'Malaysia',
-    imageUrl:
-      'https://res.cloudinary.com/intellectfox/image/upload/v1610379365/fe/the-andaman-resort_d2xksj.jpg',
-  },
-];
 
-const renderCards = function (properties) {
-  this.id = properties.id;
-  this.name = properties.name;
-  this.city = properties.city;
-  this.country = properties.country;
-  this.imageUrl = properties.imageUrl;
-  this.parentElement = document.querySelector('#render_section');
-  const card = document.createElement('figure');
-  card.classList.add('homes_pic');
-  card.innerHTML = `
-    <img src=${imageUrl} alt=${name}>
+const parent = document.querySelector('#render_section');
 
+const url = 'https://fe-student-api.herokuapp.com/api/hotels/popular';
+
+async function fetchData() {
+  const response = await fetch(url)
+  const data = await response.json()
+  console.log(data)
+  const cards = data.map(item => `
+    <img src=${item.imageUrl} alt=${item.name}>
                     <figcaption class="homes_desc">
-                        ${name}
+                        ${item.name}
                     </figcaption>
-
                     <figcaption class="homes_desc_two">
-                        ${city}, ${country}
-                    </figcaption>`;
+                        ${item.city}, ${item.country}
+                    </figcaption>
+  `
+  );
+  parent.insertAdjacentHTML('afterbegin', cards);
+
+}
 
-  parentElement.append(card);
-};
-data.forEach((item) => renderCards(item));
+fetchData()
```

Comment:
fix css, at least add normal sizes for photos please
<img width="1440" alt="Screenshot 2022-07-16 at 21 04 00" src="https://user-images.githubusercontent.com/25488411/179364907-6ca5404f-f28b-4283-b52e-0942a72575c7.png">